src/storage/FileLoader.cpp

```diff
@@ -30,6 +28,30 @@ namespace storage {
 #define WAMR_AOT_FILENAME "function.aot"
 #define SGX_WAMR_AOT_FILENAME "function.aot.sgx"
 
+static std::uintmax_t removeAllInside(const std::filesystem::path& dir)
+{
+    std::uintmax_t removedItemsCount{ 0 };
+
+    if (not is_directory(dir)) {
```

Comment:
We use `!`, not `not` in general. Would be good to stay consistent with the repo style.

Where is this `is_directory` coming from? Is it in `std::filesystem`? If so, we should prepend `std::filesystem` (as is done elsewhere in this file).

Reply:
This is probably argument-dependent lookup (ADL) - dir's type is from std::filesystem:: so it gets included in the search path - I wonder if there's a clang-tidy lint to find these lookups automatically in case one slips in somewhere